docs/_posts/2016-01-11-admin-manual.md

```diff
@@ -173,10 +173,7 @@ The configuration files are:
 | `submission.properties`   | properties to set the submission environment.              |
 
 ## Security Environment
-MISO can use either LDAP or JDBC as an authentication mechanism. The mechanism
-is set in both `/storage/miso/security.properties` and the
-`$CATALINA_HOME/bin/setenv.sh` or `/etc/default/tomcat8` files and both must be
-the same.
+MISO can use either LDAP or JDBC as an authentication mechanism.
```

Comment:
If this is the case, then it would make more sense to move the entire `-DsecurityMethod` part down here rather than having the user add it and then have to go looking upwards and re-open the file.